pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue

```diff
@@ -109,6 +140,37 @@ export default defineNuxtComponent({
         ['Admin Discount', this.rawBooking.adminDiscountPercentage * 100 + '%']
       ];
     }
+  },
+  methods: {
+    anyTicketsChecked(booking) {
+      for (const ticket of booking.tickets) {
+        if (ticket.checkedIn) {
+          return true;
+        }
+      }
+      return false;
+    },
+    checkedInShow(ticket) {
+      if (ticket.checkedIn) {
+        return 'Yes';
+      } else {
+        return 'No';
+      }
+    },
+    checkedInByDisplay(ticket) {
+      if (ticket.checkedIn) {
+        return ticket.checkedInBy.firstName + ' ' + ticket.checkedInBy.lastName;
+      } else {
+        return 'N/A';
+      }
+    },
```

Comment:
Also just a general heads up here that in this situation the `else` is redundant. Typically else statements aren't actually used in programming that much, because there is usually always a way to make it so you don't need it!

Reply:
These inline functions are really neat: I didn't know they existed! `checkedInShow` and `checkedInByDisplay` are now moved in, but I think `checkedInAtDisplay` has to stay out because it calls on `DateTime` and errors out if I try to move it in. There's maybe a nice way to fold it in (or `import` inline too)?

`else` is actually used here, because the `Checked In By` and `Checked In At` columns fold out if any ticket is checked in. As a result, you can get these situations, which I think the `else` catches?
![image](https://user-images.githubusercontent.com/25545513/230346130-5dfce243-4a06-4ebf-bebf-baf58f961a1b.png)

Reply:
Happy to leave `checkedInAtDisplay` as you have done. There is a way to get imports in the template, but it's a bit whack.

On the else side, you could have done this to get rid of the else (I didn't mean getting rid of the actually logic, just that an `else` operation is implied based on you using a `return` inside the `if` statement):
```js
if (ticket.checkedIn) {
    return ticket.checkedInBy.firstName + ' ' + ticket.checkedInBy.lastName;
}
return 'N/A';
```

But as you've now done even this can be simplified into one line 😃